addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10671,7 +10671,25 @@ msgctxt "#19296"
 msgid "No PVR add-on enabled"
 msgstr ""
 
-#empty strings from id 19297 to 19498
+#. Label for filtered recordings view value: 'show all'
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19297"
+msgid "All recordings"
+msgstr ""
+
+#. Label for filtered recordings view value: 'show only unwatched'
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19298"
+msgid "Unwatched"
+msgstr ""
+
+#. Label for filtered recordings view value: 'show only watched'
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19299"
+msgid "Watched"
```

Comment:
@MartijnKaijser on second thought, this is bullsh**. I will improve the implementation not to rely any longer on three num values in a row. ;-)